industry_benchmarks/utils/fix_networks.py

```diff
@@ -420,17 +444,17 @@ def get_fixed_alchemical_network(ducktape_network, alchemical_network):
     Create an alchemical networks with only the missing edges.
     """
     solv = openfe.SolventComponent()
-
+    cofactors = []
     for node in alchemical_network.nodes:
         prot_comps = [isinstance(comp, ProteinComponent) for comp in node.components.values()]
```

Comment:
This is a bug (in the main script), it leads to not saving out the `ProteinComponent` and also not necessarily the cofactors. I added a suggestion in the main PR, could you maybe add a test for reading in one of the complex .json files that this script produces and check that it has `SmallMoleculeComponent`, `ProteinComponent`, solvent, and cofactor (if you use eg5)?